syntax = "proto3";

package messaging;

message VICON {
    float pos_x=1;
    float pos_y=2;
    float pos_z=3;

    float quat_w=4;
    float quat_x=5;
    float quat_y=6;
    float quat_z=7;

    double time=8;
}